### PR TITLE
Add Next.js News

### PR DESCRIPTION
**Pull request by vvo**
Example: https://nextjsnews.com/2

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -99,6 +99,7 @@ Thanks to all [contributors](https://github.com/zudochkin/awesome-newsletters/gr
 - [TypeScript Weekly](https://www.typescript-weekly.com/). The best TypeScript links every week, right in your box.
 - [JSter](http://jster.net/blog). No nonsense list of curated JavaScript links to your mailbox once every two weeks. [Subscribe](https://us7.list-manage.com/subscribe?u=ed40c0084a0c5ba31b3365d65&id=ec6f32bf5e).
 - [Best of JavaScript](https://weekly.bestofjs.org/) Weekly rankings about the most popular open-source projects related to Node.js and the web platform.
+- [Next.js News](https://nextjsnews.com). Next.js focused newsletter sent every month. Contains reviews of upcoming features, best articles, tools, and plugins.
```
Comment (ferhatelmas) [inline README.md:102]: looks very good suggestion but having it in react section would be more logical.

Comment (vvo) [reply README.md:102]: thanks, updating now

Comment (vvo) [reply README.md]: done

**Review by ferhatelmas — APPROVED** [review]
LGTM

**Comment by ferhatelmas** [comment]
@vvo thanks for your contribution :heart:

**Pull request merged**